src/pkg/services/m365/api/graph/service.go

```diff
@@ -392,11 +393,12 @@ func (aw *adapterWrap) Send(
 			break
 		}
 
+		err = stackWithCoreErr(ictx, err, 1)
```

Comment:
is this going to cause additional core errors to be added to the stack since from my understanding it's above the the stuff defined in in http_wrapper.go?